### PR TITLE
Slow loading of HR recruitment page on the Portal

### PR DESCRIPTION
**Pull request by Harshvardhan80**
Targets #3113 

### Description
When I used to open the HR recruitment page, it would take more than 10 seconds to load. I would like to improve its performance.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/HR/Http/Controllers/Recruitment/ApplicationController.php [hunk 1]
```diff
@@ -71,6 +71,13 @@ public function index()
             'should_skip_page' => false,
         ]);
 
+        $hr_job_id = request()->has('hr_job_id') ? '&hr_job_id=' . request('hr_job_id') : '';
```
Comment (LavishRawat) [inline Modules/HR/Http/Controllers/Recruitment/ApplicationController.php:74]: @Harshvardhan80 can we fix the naming convention here? Use camelCase.

File: Modules/HR/Http/Controllers/Recruitment/ApplicationController.php [hunk 2]
```diff
@@ -71,6 +71,13 @@ public function index()
...
+        $graduation_year = request()->has('end-year') ? '&end-year=' . request('end-year') : '';
```
Comment (LavishRawat) [inline Modules/HR/Http/Controllers/Recruitment/ApplicationController.php:75]: @Harshvardhan80 same here. Fix the naming convention in all places.

**Review by LavishRawat — CHANGES_REQUESTED** [review]

File: Modules/HR/Http/Controllers/Recruitment/ApplicationController.php (hunk 2 above)
Comment (Harshvardhan80) [reply Modules/HR/Http/Controllers/Recruitment/ApplicationController.php]: yes, sure

File: Modules/HR/Http/Controllers/Recruitment/ApplicationController.php (hunk 1 above)
Comment (Harshvardhan80) [reply Modules/HR/Http/Controllers/Recruitment/ApplicationController.php]: > yes, sure

**Review by Ayush8923 — APPROVED** [review]
Looks good..

**Review by LavishRawat — APPROVED** [review]

**Pull request merged**